kvbc/test/sparse_merkle_storage/db_adapter_unit_test.cpp

```diff
@@ -1189,6 +1200,135 @@ TEST_P(db_adapter_custom_blockchain, delete_genesis_and_verify_stale_index) {
   ASSERT_FALSE(hasProvableStaleIndexKeysSince(adapter.getDb(), 2));
 }
 
+TEST_P(db_adapter_custom_blockchain, delete_genesis_and_verify_stale_key_deletion_for_non_provable_keys) {
```

Comment:
I think it might make sense to write this test without any provable (tree) keys as we already have such a test. Using non-provable keys only will simplify it.

Reply:
Done: removed the provable keys.